server/database/controllers/index.ts

```diff
@@ -0,0 +1,86 @@
+const models = require('../models/index.ts');
+
+type UserData = {
+  userName: string;
```

Comment:
I will change things on my end so that userName is firstName + ' ' + lastName. 

ex. userName = Joshua Waalkes

Reply:
Are you asking for first + last name in the sign up process? I guess i wasnt even considering that data. I was thinking username could be whatever they want -- like Nouse41 or JoshuaWaalkes, but doesn't necessarily have to be first+last. I could add first/last names to the user table too if that helps